openverse_catalog/dags/common/loader/reporting.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+import common
+
+
+logging.basicConfig(
+    format="%(asctime)s - %(name)s - %(levelname)s:  %(message)s", level=logging.INFO
+)
+logger = logging.getLogger(__name__)
+logging.getLogger(common.urls.__name__).setLevel(logging.WARNING)
```

Comment:
The logging config steps aren't needed here (more on that in #299). We just need the following:

```suggestion
logger = logging.getLogger(__name__)
```